composeml/label_times.py

```diff
@@ -226,3 +226,45 @@ def bin(self, bins, quantiles=False, labels=None, right=True):
 
         label_times.transforms.append(transform)
         return label_times
+
+    def sample(self, n=None, frac=None, random_state=None):
+        """
+        Return a random sample of labels.
+
+        Args:
+            n (int or dict) : Sample labels by count. A dictionary returns
+                the number of samples per label. Cannot be used with frac.
+            frac (float or dict) : Sample labels by fractions. A dictionary returns
+                the fraction of the count per label. Cannot be used with n.
+            random_state (int) : Seed for the random number generator.
+
+        Returns:
+            LabelTimes : Random sample of labels.
```

Comment:
can we add examples to the docstring like we do the other transforms?

Reply:
yes, added examples